### DISCUSSION TITLE
Move paper on bulk_extractor and memory analysis to a findable location

### DISCUSSION THREAD

**Discussion by fcgreg** [simsong/bulk_extractor#425] [Show and tell]
Greetings all,
Sorry I've been quiet since last year. I had intended to make more progress on rewriting the BE Viewer to a modern Java runtime but I took on some additional assignments at work--delays have been plentiful. I'll try to help out more in the near future.

In the meantime, last year I wrote a paper/guide on installing and using BE 1.5.x for rapidly triaging memory acquisitions. It contains up-to-date info on installing and using BE and BE Viewer in a Windows environment. Would this be helpful to the community here, or would it be better to host on my own repositories elsewhere?  In the meantime, I'll attach it here for your review.
~ Greg
[Bulk_Extractor and Memory Analysis-rev1.2.pdf](https://github.com/simsong/bulk_extractor/files/12566935/Bulk_Extractor.and.Memory.Analysis-rev1.2.pdf)

**Comment by simsong** [comment]
Hi Greg. Sorry for the delay in getting back to you on this. 
This is a great paper. We can link to it from the Digital Corpora website. Or, if you wish, we can turn it into a web page / tutorial there. What is your preference?

**Reply by fcgreg** [reply]
Thanks for the reply and I'm sorry I didn't respond for so long.
I am open to any option you believe is helpful. My initial thoughts are the linking to it from the Digital Corpora website in its PDF format would be the easiest. I also update it periodically and am happy to provide updates with each revision.

**Comment by simsong** [comment]
I would rather just put it on the digitalcorpora website, if that is okay. We can update it later. Would that work for you?

**Reply by fcgreg** [reply]
Yes, that sounds good to me. Thanks.

**Comment by simsong** [comment]
Actually, I'll add it to the BE doc directory.

**Reply by fcgreg** [reply]
I see that change now. Looks good to me, thanks.

**Comment by simsong** [comment]
Thanks for your help!
